packages/devextreme/js/core/utils/license.js

```diff
@@ -0,0 +1,43 @@
+import { parseToken } from '../../__internal/core/license/license_validation';
```

Comment:
It's better to develop a new code with TS
We already have a directory in ```js/__internal``` for this feature: [link](https://github.com/DevExpress/DevExtreme/tree/23_2/packages/devextreme/js/__internal/core/license)

Also, let's use types from the ```packages/devextreme/js/__internal/core/license/license_validation.ts```
(See comments below)